source/deployment/cluster.rst

```diff
@@ -134,75 +134,143 @@ A sample configuration for NGINX is provided below.  It assumes you have two Mat
+6. Start one of the Mattermost servers.
+7. When the server is running, start the other servers.
+8. Restart NGINX.
 
 Troubleshooting
-~~~~~~~~~~~~~~~~~~~~~~~
+---------------
 
 Red Server Status
----------------------------
-When high availability is enabled, the System Console displays the server status as red or green, indicating if the servers are communicating correctly with the cluster. The servers use inter-node communication to ping the other machines in the cluster, and once a ping is established the servers exchange information, such as server version and configuration files. Red server status may display for the following reasons:
+~~~~~~~~~~~~~~~~~
+
+When high availability is enabled, the System Console displays the server status as red or green, indicating if the servers are communicating correctly with the cluster. The servers use inter-node communication to ping the other machines in the cluster, and once a ping is established the servers exchange information, such as server version and configuration files.
+
+A server status of red can occur for the following reasons:
 
 - **Configuration file mismatch**: Mattermost will still attempt the inter-node communication, but the System Console will show a red status for the server since the high availability feature assumes the same configuration file to function properly.
-- **Server version mismatch**: Mattermost will still attempt the inter-node communication, but the System Console will show a red status for the server since the high availability feature assumes the same version of Mattermost is installed on each server in the cluster. It is recommended to use the `latest version of Mattermost <https://www.mattermost.org/download/>`_ on all servers. Follow the `upgrade procedure <https://docs.mattermost.com/administration/upgrade.html>`_ for any server that needs to be upgraded.
-- **Server is down**: If an inter-node communication fails to send a message it will attempt again in 15 seconds.  If the second attempt fails, the server is assumed to be down. An error message is written to the logs and the System Console will show a status of red for that server.
+- **Server version mismatch**: Mattermost will still attempt the inter-node communication, but the System Console will show a red status for the server since the high availability feature assumes the same version of Mattermost is installed on each server in the cluster. It is recommended to use the `latest version of Mattermost <https://www.mattermost.org/download/>`_ on all servers. Follow the upgrade procedure in :doc:`../administration/upgrade` for any server that needs to be upgraded.
+- **Server is down**: If an inter-node communication fails to send a message it will attempt again in 15 seconds. If the second attempt fails, the server is assumed to be down. An error message is written to the logs and the System Console will show a status of red for that server.
```

Comment:
The system will continue to ping the down system every 15 seconds and if successful send any new messages.  It will only attempt to send a messages twice before dropping it.